src/Security/Authentication/Certificate/src/Microsoft.AspNetCore.Authentication.Certificate.csproj

```
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>ASP.NET Core middleware that enables an application to support certificate authentication.</Description>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <DefineConstants>$(DefineConstants);SECURITY</DefineConstants>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageTags>aspnetcore;authentication;security;x509;certificate</PackageTags>
```

```suggestion
    <PackageTags>aspnetcore;authentication;security;x509;certificate</PackageTags>
    <IsShippingPackage>true</IsShippingPackage>
```

If you want this to ship to nuget.org, add this property. Otherwise, this package will be considered internal-only (which is okay if you're still in the early stages of building out this API)